Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -123,77 +123,115 @@ The list should be maintained in alphabetic order. DELETE This Row
 
-* **What problem does this solve?**
-
-This solves the problem of incompatible and opaque metrics that are potentially _gameable_ today making it difficult for the end-consumer to make a choice that is aligned with what they are looking for in terms of meeting their environmental goals when purchasing / using software solutions. It will also help developers and users compare one software solution offering against another on environmental impacts.
-
-* **How can this specification be applied?**
+#### Applications Of This Specification
 
 The specification can be applied to any software to measure and reduce its carbon emissions by creating a standardized and practical methodology. 
 
-* **Who is the target audience?** 
+#### Target Audience
 
 The target audience for this are technical stakeholders (e.g. software architects, developers, and maintainers) who ideally can use this as a methodology so that they can understand the characteristics of their software solution and minimize the associated emissions.
 
-This standard uses a consequential approach to calculating the real world emissions associated with software. This approach measures the total change in global emissions associated with a particular piece of software.
+
+## Methodology Summary 
+
+This standard can be used to calculate the real-world emissions associated with software by measuring the total change in global emissions associated with a particular piece of software.
 
 Electricity has a carbon intensity depending on where and when it is consumed. An intensity is a rate. It has a numerator and a denominator. A rate provides you with helpful information when considering the growth of a software product and allows for the computation of a marginal rate.
 
-A carbon intensity is a rate of carbon emissions compared against another measure. 
+To calculate the carbon intensity we need the following:
 
-To calculate the carbon intensity we need to decide two things.
+`O`= Operational emissions of a given piece of software
 
-`C / R`
+`E` = Embodied emissions of a given piece of software
 
-`C` = The total amount of carbon the software is emitting over a time period.
+**These are used to calculate total carbon emissions (`C`):**
 
-`R` = The other measure we will be comparing against.
+`C` = Total amount of carbon the software is emitting over a time period.
 
-## Taxonomy of Green Software
+`C = O + Embodied `
```

Comment:
`C  = O + E` to be consistent with the text previos

Reply:
Moving a few steps ahead, I would say our methodology is to get to:

`CI = (O + E) / R`

Where R is a denominator that makes sense for your application, e..g if you are Windows it might be per Device or per User. If you are an API it might be per API call?